src/yellow/components/shape/_index.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../../../node_modules/@getninjas/niten-tokens/build/web/niten-tokens';
+
+.shape {
+  border-color: $color-interface-shade-dark;
+  border-radius: $size-border-radius-s;
+  border-style: solid;
+  border-width: 1px;
+  box-sizing: border-box;
+  transition: border .3s ease-in-out;
+
+  &:hover {
+    border-color: $color-brand-first-extra-dark;
+    border-width: 2px;
+  }
+
+  &--clickable {
+    box-shadow: $shadow-bottom-level-1;
+    cursor: pointer;
+  }
+
+  &--positive {
+    border-color: $color-interface-positive-dark;
```

Comment:
It seems that is possible to use the `&` selector to merge this rules:
```
&--positive {
  &,
  &:hover {
    border-color: $color-interface-positive-dark;
  }
}
```

That way we don't repeat the same rule for 2 selectors. I tested in a codepen and worked.

Reply:
Yes it really does.

But my fear is that it will not be so clear to other people what is happening if we do it that way. What do you think about this?